bindings/rascal/neighbourlist/structure_manager.py

```diff
@@ -116,19 +144,18 @@ def sanitize_non_periodic_structure(structure):
 
     if np.all(structure['pbc'] == 0):
         cell = structure['cell']
-        if np.allclose(cell,np.zeros((3,3))):
+        if np.allclose(cell, np.zeros((3, 3))):
             pos = structure['positions']
-            bounds = np.array([pos.min(axis=1),pos.max(axis=1)])
+            bounds = np.array([pos.min(axis=1), pos.max(axis=1)])
             bounding_box_lengths = bounds[1]-bounds[0]
             new_cell = np.diag(bounding_box_lengths)
             CoM = pos.mean(axis=1)
             disp = 0.5*bounding_box_lengths - CoM
-            new_pos = pos + disp[:,None]
+            new_pos = pos + disp[:, None]
```

Comment:
unrelated, but we should maybe enforce that `make pretty` is ran on each PR, else it introduces noise on unrelated PR.

Reply:
That is the rule but it is hard enforce all the time I guess.

Reply:
We could test for it in CI. I added this point for the next meeting.